environment.yml

```
@@ -8,5 +8,5 @@ dependencies:
  - dask>=2.6
  - cftime
  - python-dateutil>=2.8.1
- - cf_xarray>=0.3.1 # Note that conda-forge package is cf_xarray (!= cf-xarray)
+ - cf_xarray>=0.3.1,<=0.8.4 # Note that conda-forge package is cf_xarray (!= cf-xarray)
```

See issue about cf_xarray in `requirements.txt`

---

Let's let conda figure this out on its own:
```suggestion
 - cf_xarray>=0.3.1 # Note that conda-forge package is cf_xarray (!= cf-xarray)
```